python/src/BetaMuSigma_doc.i.in

```diff
@@ -1,6 +1,9 @@
 %feature("docstring") OT::BetaMuSigma
 "Beta distribution parameters.
 
+Available constructors:
+    BetaMuSigma(*mu=0.5, sigma=0.223607, a=0.0, b=1.0*)
```

Comment:
no need to documents ctors when only one is available, the default values can be mentionned in the parameter list below